feature_annotations.yaml

```diff
@@ -0,0 +1,24 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
+
+source_path: ./
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 50.0
+annotations:
+  feature_toggle:
+    - ".. feature_toggle_name:":
+    - ".. feature_toggle_type:":
```

Comment:
```
feature_toggle_use_cases: (Future?  Just part of description?  Other?)
feature_toggle_implementation (name?): choices: [ConfigurationModel, WaffleFlag, WaffleSwitch, DjangoSetting]    
feature_toggle_type: choices: [switch, rollout, group]
```
FYI: @albemarle @nasthagiri @pdpinch

Reply:
I like `feature_toggle_type`.  Maybe we can use `feature_` instead of `feature_toggle_` as the basename for all of these?  Then the implementation one could be `feature_implemented_as` or something.